hive-plugins/src/main/java/co/cask/plugin/etl/batch/commons/HiveSchemaConverter.java

```diff
@@ -88,7 +91,7 @@ private static PrimitiveTypeInfo getType(String name, Schema.Type category) {
         return TypeInfoFactory.binaryTypeInfo;
       default:
         throw new IllegalArgumentException(String.format(
-          "Schema contains field '%s' with unsupported type %s", name, category.name()));
+          "Schema contains field '%s' with unsupported type %s", name, type));
```

Comment:
What's the corrective action one can take if this problem happens and why would it happen.